libkbfs/folder_branch_ops.go

```diff
@@ -3233,9 +3368,13 @@ func (fbo *folderBranchOps) undoMDUpdatesLocked(ctx context.Context,
 				fbo.getCurrMDRevisionLocked(lState)}
```

Comment:
I meant that we redundantly check that the version of the MD is one less than the previous one. Added a TODO to just rely on the same check in `setHeadPredecessorLocked`